src/vcpkg/tools.cpp

```diff
@@ -463,6 +487,31 @@ namespace vcpkg
         }
     };
 
+    struct AzCliProvider : ToolProvider
+    {
+        virtual bool is_abi_sensitive() const override { return false; }
+        virtual StringView tool_data_name() const override { return Tools::AZCLI; }
+        virtual std::vector<StringView> system_exe_stems() const override { return {Tools::AZCLI}; }
+        virtual std::array<int, 3> default_min_version() const override { return {2, 30, 0}; }
```

Comment:
What do you specifically need in 2.30 given that the comment below strongly suggests you only tested with 2.64?

Reply:
I've only tested it with az cli version 2.64. Not exactly sure what minimum version to add here, I see the minimum version for `mono` is version `0.0.0`.

Reply:
Where did 2.30 come from then?